behat-tests.yml

```diff
@@ -0,0 +1,32 @@
+default:
+  suites:
+    default:
+      contexts:
+        - FeatureContext
+        - Behat\MinkExtension\Context\MinkContext
+        - PaulGibbs\WordpressBehatExtension\Context\ContentContext
+        - PaulGibbs\WordpressBehatExtension\Context\DashboardContext
+        - PaulGibbs\WordpressBehatExtension\Context\SiteContext
+        - PaulGibbs\WordpressBehatExtension\Context\UserContext
+        - PaulGibbs\WordpressBehatExtension\Context\WordpressContext
+
+  extensions:
+    Behat\MinkExtension:
+      base_url: http://wordpress.dev
+      default_session: default
+      javascript_session: selenium2
+      sessions:
+        default:
+          goutte:
+            guzzle_parameters:
+              verify: false  # Allow self-signed SSL certificates
+        selenium2:
+          selenium2: ~
+
+    PaulGibbs\WordpressBehatExtension:
+      default_driver: wpapi
```

Comment:
Yes - but I think we should try to keep parity amongst the drivers as much as possible. In that vein it would be good to run the tests for each driver (or at least `wpcli` and `wpapi`, which I suspect will be the most common).